src/router/index.js

```diff
@@ -24,4 +30,9 @@ const router = new VueRouter({
   routes,
 });
 
+// eslint-disable-next-line arrow-parens
+router.afterEach(to => {
+  document.title = `GorillaCast - ${to.name}`;
```

Comment:
I recommend using pipe separator `Gorilla Cast | ${to.name}``

Reply:
done